arrow/arrow.py

```diff
@@ -628,6 +661,10 @@ def replace(self, **kwargs):
 
             if key in self._ATTRS:
                 absolute_kwargs[key] = value
+            elif key == "fold":
+                # datetime replace will not work in 2.7/3.5
```

Comment:
I would put a TODO comment to come back to this once we drop Python 2.7 and 3.5.